libs/@local/hash-validation/src/lib.rs

```diff
@@ -130,6 +169,47 @@ mod tests {
         id: VersionedUrl,
     }
 
+    impl EntityProvider for Provider {
+        async fn provide_entity(
+            &self,
+            entity_id: EntityId,
+        ) -> Result<&Entity, Report<InvalidEntity>> {
+            self.entities
+                .get(&entity_id)
+                .ok_or_else(|| Report::new(InvalidEntity { id: entity_id }))
+        }
+    }
+
+    impl EntityTypeProvider for Provider {
+        async fn is_parent_of(
+            &self,
+            child: &VersionedUrl,
+            parent: &VersionedUrl,
+        ) -> Result<bool, Report<InvalidPropertyType>> {
```

Comment:
same applies below
```suggestion
        ) -> Result<bool, Report<InvalidEntityType>> {
```

Reply:
Thanks, I fixed this in https://github.com/hashintel/hash/pull/3534/commits/45c2d1a7ec00b3c95010886dbc597ac7d0361bcc